components/feature/readerview/src/main/res/values/strings.xml

```diff
@@ -6,18 +6,32 @@
   -->
 
 <resources>
-    <string name="mozac_feature_readerview_sans_serif">Sans serif</string>
+    <!-- Name for one of the font options available to use -->
+    <string name="mozac_feature_readerview_sans_serif_font">Sans serif</string>
+    <!-- Accessible description for the font option -->
     <string name="mozac_feature_readerview_sans_serif_font_desc">Sans Serif font</string>
-    <string name="mozac_feature_readerview_serif_desc">Serif</string>
+    <!-- Name for one of the font options available to use -->
+    <string name="mozac_feature_readerview_serif_font">Serif</string>
+    <!-- Accessible description for the font option -->
     <string name="mozac_feature_readerview_serif_font_desc">Serif font</string>
-    <string name="mozac_feature_readerview_negative_sign">-</string>
+
+    <string name="mozac_feature_readerview_negative_sign" translatable="false">-</string>
```

Comment:
translatable="false": this won't work with our tooling. This should go in a separate file, such as non_localizable_strings.xml (which exists for other components).